### PR TITLE
added langtrace

### PR DESCRIPTION
**Pull request by yemiadej**
## Requirements

 * [X] This is Software as a Service not self hosted
 * [X] It has a free tier not just a free trial
 * [X] Pricing information is clearly visible without signup or phone calls
 * [X] The submission mentions what is free
 * [X] The submission is not already present in the list
 * [X] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1093,6 +1093,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Findr](https://www.usefindr.com/) - Universal search that lets you search all your apps, at once. Search assistant that lets you answer questions using your information. Free plan offers unlimited unified search and 5 co daily co pilot queries.
   * [ReportGPT](https://report-gpt.io) - AI Powered Writing Assistant. The entire platform is free as long as you bring your own API key.
   * [Clair](https://askclair.ai/) - Clinical AI Reference. Students have free access to the professional tool suite, which includes Open Search, Clinical Summary, Med Review, Drug Interactions, ICD-10 Codes, and Stewardship. Additionally, a free trial for the professional suite is available.
+  * [Langtrace](https://langtrace.ai) - enables developers to trace, evaluate, manage prompts and datasets, and debug issues related to an LLM application’s performance. It creates open telemetry standard traces for any LLM which helps with observability and works with any observability client. Free plan offers 50K traces/month.
```
[the remark this answers is not in the document]
Comment (yemiadej) [reply README.md:1096]: We're not charging anything for now. We will always have a free tier of 50K traces when we do start charging.

[the remark this answers is not in the document]
Comment (ripienaar) [reply README.md:1096]: I would have to see this written somewhere on the site to be able to verify this

[the remark this answers is not in the document]
Comment (yemiadej) [reply README.md:1096]: Please check out docs 

https://docs.langtrace.ai/introduction#faqs

[the remark this answers is not in the document]
Comment (ripienaar) [reply README.md:1096]: Based on that, there is no ongoing commitment to being free, no detail about the number of requests etc.

Screenshot 2024-07-30 at 18 25 02

So the conclusion is that it's just not a good fit for the list.

[the remark this answers is not in the document]
Comment (yemiadej) [reply README.md:1096]: No worries. We'll revisit once our website and docs are updated. Thanks.

**Pull request closed without merging**